### ISSUE TITLE
Consider adding gulp build tools and bower to the codebase

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/STNWeb2.0#19]
@troddick It looks like STNWeb2.0 does not have any build system yet. We should be using gulp to build a release build and do browser debugging with auto refresh, and bower to pull in dependencies. Any objection to me adding these?

**Comment by troddick** [comment]
I added Bower.

**Comment by troddick** [comment]
sorry.. didn't read the actual comment, just the heading. 

I did add bower, but feel free to integrate gulp. I didn't get too much into all this.. readMe file and release vs debug versions and all that. Just wanted the code up so I could keep working on it.
Thanks.

**Comment by BlakeDraper** [comment]
Yeah no problem, I know you have been working hard at getting this thing going quickly. I see two references to a bower components folder, but that folder is not in the actual repo. Are you adding that folder to the web server separately? There is also no bower.json file to tell which dependencies are in the project, so another developer could not run this on their own machine without manually adding the dependencies listed in the index, so we should get all that up to standard.

**Comment by troddick** [comment]
hmm. OK. I thought when I committed from VS, it would also include the bower stuff. I have a bower folder in the STNWeb2.0 folder on my computer, and a bower.js at the same level. Let me see how I can get it on here.

**Comment by troddick** [comment]
added the bower. Those files were being excluded for some reason. Thanks
